www/docs/customization/artifactory.md

```diff
@@ -183,6 +183,10 @@ artifactories:
       ...(edited content)...
       TyzMJasj5BPZrmKjJb6O/tOtEIJ66xPSBTxPShkEYHnB7A==
       -----END CERTIFICATE-----
+
+    # Additional properties supplied as matrix parameters
```

Comment:
probably good to explain that it goes in the end of the url as `;key=value`

also need to specify this option was added in the v1.18:

```suggestion
    # Additional properties supplied as matrix parameters.
    # 
    # These values will be appended to the url as `;key=value`.
    # 
    # Since: v1.18
```